platforms/pf-robot-motion-control/motion_control.cpp

```diff
@@ -674,12 +674,12 @@ void pf_init_motion_control(void)
     // Init motor driver
     motor_driver.init();
```

Comment:
You can remove this line, as you moved the initialisation set inside the motor implementation.

Reply:
Done.